### PR TITLE
Crypto Currency Tools & Algorithms

### PR DESCRIPTION
**Pull request by kennethreitz**
✨🍰✨

https://github.com/kennethreitz/awesome-coins/

This is an awesome list for crypto currency.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -553,6 +553,7 @@
 - [Homematic](https://github.com/hobbyquaker/awesome-homematic) - Smart home devices.
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
+- [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins/)
```
Comment (sindresorhus) [inline readme.md]: Can you add a short description here about what crypto currencies are? Not about the list itself. Per guidelines.

How about this?

> Digital currency where encryption is used to regulate the generation of units and verify transfers.

**Comment by sindresorhus** [comment]
See: https://github.com/sindresorhus/awesome/pull/1033#discussion_r138175933

**Comment by sindresorhus** [comment]
Maybe link to https://en.m.wikipedia.org/wiki/Cryptocurrency

**Comment by sindresorhus** [comment]
The list looks pretty useful by the way.

**Comment by sindresorhus** [comment]
> Each link description starts with an uppercase character and ends with a period.
Example: - [AVA](…) - JavaScript test runner.

From the guidelines ⬆️

**Comment by kennethreitz** [comment]
Thanks for the feedback. Will update.

**Comment by kennethreitz** [comment]
- Link to wikipedia added
- License added
- Code of conduct added.
- Contribution guidelines added.

**Comment by kennethreitz** [comment]
I don't think the "ends with a period" rule applies to this repo — is it possible to make an exception?

File: readme.md (hunk 1 above)
Comment (kennethreitz) [reply readme.md]: done

**Comment by kennethreitz** [comment]
Okay, I understand now. Updated and ready!

**Comment by kennethreitz** [comment]
✨🍰✨

**Comment by sindresorhus** [comment]
🌟🎂🌟

**Pull request merged**